### PR TITLE
Bug 1118311 - Fix dragging expanded groups. r=kgrandon

### PR DESCRIPTION
**Pull request by Cwiiis**

### CODE REVIEW

**Comment by try-server-hook** [comment]
[Cwiiis Chris Lord (Cwiiis)](https://github.com/Cwiiis) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=25fca8aedde6)

**Comment by try-server-hook** [comment]
[Cwiiis Chris Lord (Cwiiis)](https://github.com/Cwiiis) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=e23227a2c017)

**Comment by try-server-hook** [comment]
[Cwiiis Chris Lord (Cwiiis)](https://github.com/Cwiiis) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=29bfde9d8457)

File: apps/verticalhome/test/marionette/group_drag_expanded_test.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    client.apps.launch(home.URL);
+    home.waitForLaunch();
+  });
+
+  test('check that dragging an expanded group collapses correctly', function() {
+    function getRect(el) {
+      return el.getBoundingClientRect();
+    }
+
+    var groupBackgroundSelector = '#icons .group .background';
```
Comment (KevinGrandon) [inline apps/verticalhome/test/marionette/group_drag_expanded_test.js]: nit: I would opt to move this into the selectors object of verticalhome.js.

https://github.com/mozilla-b2g/gaia/blob/master/apps/verticalhome/test/marionette/lib/verticalhome.js#L31

**Comment by try-server-hook** [comment]
[Cwiiis Chris Lord (Cwiiis)](https://github.com/Cwiiis) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=44bcc787b6e7)

**Comment by try-server-hook** [comment]
[Cwiiis Chris Lord (Cwiiis)](https://github.com/Cwiiis) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=dc60f7cc3371)

**Pull request merged**